OpenTrons/opentrons-api
# refactor(opentrons-ai-client): refactoring api call and inputprompt component

# Overview
refactoring api call and input prompt component
add a new package for testing axios

close AUTH-402


# Test Plan
1. `make teardown-js && make setup-js`
2. `make -C opentrons-ai-client dev`
3. open localhost:5173 (the port must be 5173 since the backend allows us to use only 5173 port)
4. log in (if you don't have an account, you will need to sign up)
5. send a prompt
check the app displays your prompt and gets the return from OpenAI API


# Changelog
- add custom hook for callApi and get access token
- export a function as a util function
- update paragraph text style to align the font styles


# Review requests



# Risk assessment
low
